app/scripts/views/sign_in.js

```diff
@@ -18,6 +17,7 @@ define(function (require, exports, module) {
   const MigrationMixin = require('views/mixins/migration-mixin');
   const PasswordMixin = require('views/mixins/password-mixin');
   const PasswordResetMixin = require('views/mixins/password-reset-mixin');
+  const { preventDefaultThen, t } = require('views/base');
```

Comment:
We still need it in [_suggestSignUp](https://github.com/mozilla/fxa-content-server/blob/af1ce155e92c59243281a6bf25cb905a2902bd32/app/scripts/views/sign_in.js#L169). Leaving as is!